dbaas/psmdb/config.go

```diff
@@ -622,6 +622,10 @@ func (cr *PerconaServerMongoDB) SetNew(clusterName, rsName string, c dbaas.Clust
 		return errors.Wrap(err, "new replset")
 	}
 
+	if len(c.PSMDB.BrokerInstance) > 0 {
+		cr.ObjectMeta.Annotations["broker-instance"] = c.PXC.BrokerInstance
```

Comment:
1. make(map[string]string)?
2. **PXC** ? This is why I don't like current design where pxc and psmdb are put together, passed around and everybody knows about everything. That's a huge mess and leads to such kind of errors.